tpc/Makefile

```diff
@@ -122,6 +126,20 @@ else ifneq ($(findstring .tcks,$(2)),)
 	$(p1)$(cks_generator) gen-java $$< serializers.cks.$(basename $(2)) build/gen/$(2)/serializers/cks/$(basename $(2))
 else ifneq ($(findstring .avro,$(2)),)
 	$(p1)$(avro_generator) compile schema $$< build/gen/$(2)
+else ifneq ($(findstring .dsl,$(2)),)
+	rm -rf dslplatform_client_tmp java_client
+	mkdir -p dslplatform_client_tmp
+	$(p1)$(dslplatform_generator) \
+		-temp=dslplatform_client_tmp \
+		-dsl=schema \
+		-settings=manual-json,no-jackson \
+		-namespace=serializers.dslplatform \
+		-target=java_client \
+		-no-prompt \
+		-compiler=lib/dsl-compiler.exe \
```

Comment:
Currently dsl clc will download it to temp path and reference it there. So yeah... we can exclude it from lib folder